config/core.extension.yml

```diff
@@ -46,5 +46,6 @@ theme:
   classy: 0
   bartik: 0
   seven: 0
+profile: config_installer
```

Comment:
@tauno You mentioned once that config_installer isn't really a profile. Could we say something like "standard" or even "lightning" here and still have the flow work?